frontend/src/components/mine/ContactInfo/ViewPartyRelationships.js

```diff
@@ -85,11 +85,8 @@ export class ViewPartyRelationships extends Component {
   onSubmitAddPartyRelationship = (values) => {
     const payload = {
       mine_guid: this.props.mine.mine_guid,
-      party_guid: values.party_guid,
       mine_party_appt_type_code: this.state.selectedPartyRelationshipType,
-      related_guid: values.related_guid,
-      start_date: values.start_date,
-      end_date: values.end_date,
+      ...values,
```

Comment:
Is there a reason you changed this to a spread? Our standard has been to move away from this and use explicit parameters.

Reply:
I didn't know that was the standard, I just did this because I added a new value to the form and just wanted it passed through. I thought it would just be easier to pass all the form values through.

Reply:
I have re-added the explicit parameter passing.

Reply:
Thanks Justin!